web/src/components/approvals.js

```diff
@@ -83,8 +45,9 @@ class ApprovalStats extends BaseQueryComponent {
               <Card.Body>
                 <Row>
                   <Col>
-                    <ChangeApprovals
-                      data={data}
+                    <Pie
```

Comment:
Could we override the palette for the approval ? I'd like to keep the red colors for negative approval and green for +1, +2, APPROVED.

Reply:
done